### PR TITLE
[Feature] Support insert resource group

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 6 / 9 (66.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|2|5|40.00%|[703, 707, 1176]|
|:large_blue_circle:|com/starrocks/catalog/ResourceGroupClassifier.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|3|3|100.00%|[]|

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

File: fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java [hunk 1]
```diff
@@ -3069,6 +3072,7 @@ private void toThriftForCommonParams(TExecPlanFragmentParams commonParams,
                     }
                 }
             }
+            LOG.info(commonParams);
```
Comment (mofeiatwork) [inline fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java:3075]: ?

**Comment by mofeiatwork** [comment]
BTW, have you ever did any performance benchmark ? I'm afraid it will not be expected.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 5 / 8 (62.50%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|2|5|40.00%|[703, 707, 1176]|
|:large_blue_circle:|com/starrocks/catalog/ResourceGroupClassifier.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|2|2|100.00%|[]|

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Comment by dulong41** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by chaoyli — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Pull request merged**